frontend/src/locales/zh.json

```
@@ -2120,7 +2283,37 @@
     "reviewApiKey": "审查 API Key（可多个）",
     "reviewApiKeyPlaceholder": "每行一个 key（sk-...），可填多个",
     "reviewApiKeyConfigured": "已配置 {{n}} 个 key，留空保持不变",
-    "reviewApiKeyHint": "每行一个 key。配置多个可分摊 Moderations 的 TPM 额度，遇到限流会自动切换到下一个 key。",
+    "reviewApiKeyHint": "每行一个 key。配置多个可轮询分摊额度，遇到限流、无效 key 或服务端错误时自动切换。",
+    "reviewRequestMode": "请求适配模式",
+    "reviewModeChat": "Chat Completions 兼容接口",
+    "reviewModeModerations": "Moderations（兼容旧配置）",
+    "reviewScope": "审核范围",
```

Biome (2.5.5)

[error] The key reviewScope was already declared.

(lint/suspicious/noDuplicateObjectKeys)